docs/source/model_design/risk_factors.rst

```diff
@@ -1,55 +1,299 @@
+
+For rare diseases, risk and rates are numerically similar. 
+
+.. todo::
+  example of how this is so
+
++------------------------+-----------------------------------------------+
+|   Measure              | Math                                          |
++------------------------+-----------------------------------------------+
+| | Incidence rate       |:math:`\frac{\text{new cases of disease during |
+| | Incidence density    |time period}}{\text{total person-time at       |
+| | Force of morbidity or|risk during time period}}`                     |
+| | mortality            |                                               |
++------------------------+-----------------------------------------------+
+
+.. todo::
+   James says: give example and how the time period needs to be specified. 
+   Has this been addressed in the examples?
+
+Relationship between rate and risk
+**********************************
+
+Risk depends on both the incidence rate and on the duration of the at-risk period. In vivarium, we apply the incidence rate to each simulant at each time step to estimate the risk of developing disease where the duration of the period is the duration of the time-step: 
+
+    Risk = :math:`1 – e^\text{( –incidence rate × duration of the period at risk)}`
+
+For disease that have a low incidence rate or when the period at risk is short, the following approximation can be used:
+
+    Risk = incidence rate × duration of the period at risk.
+
+Example: 
+    
+  * The incidence rate of a particular condition in a population is 50 per 100 000 person-years. The risk for an individual in this population of developing this condition during a five-year period (assuming no other causes of death) is given by
+
+      - 5-year risk = :math:`1 – e^\text{( –0.0005 per person-year × 5 years)}` = 0.25%
+        | The simplified equation can yield the same result
+      - 5-year risk=0.0005 per person-year X 5 years = 0.25%
+  * Consider now a common condition with an incidence rate of 300 per 1000 person-years
+
+      - 5-year risk = :math:`1 – e^\text{( –0.3 per person-year × 5 years)}` = 78%
+        | The simplified equation does not yield the same result
+      - 5-year risk = 0.3 per person-year X 5 years = 150%
+
+Measures of effect
+++++++++++++++++++
+
+Measures of effect are used to compare the frequency of outcome between specified populations. When one population group is exposed to a risk factor and the other is not, measures of effect can be used to study associations between frequency of disease and the risk factor. They reflect the increase in frequency of disease in one population in comparison with another. Frequency measures (e.g. risks, rates) can be compared by estimating their *ratios* or *differences*. 

```

Comment:
This is fantastic---I almost think this section could start off with this, with your explanations for ratio measures and difference measures.... but my inclination to do so is from a GBD-centric view of how risk factors and relative risks are conceptualized, so very open to push-back.

To me, it would then be helpful to follow these sections with the further details on risks and rates that you have above, and then nixing the prevalence and incidence sections, as we already have documentation elsewhere on those.

Reply:
ya, lets discuss as a group how to restructure this whole section!